src/Interfaces/Standalone/CMakeLists.txt

```diff
@@ -1,26 +1,11 @@
 # Cloud-J/src/CMakeLists.txt
 
-add_library(CloudJ_Standalone STATIC EXCLUDE_FROM_ALL
-	CJ77.F90
-)
-target_link_libraries(CloudJ_Standalone
-	PUBLIC CloudJ_Core
-)
-target_include_directories(CloudJ_Standalone
-        INTERFACE ${CLOUDJ_BINARY_DIR}/mod
-)
-target_compile_options(CloudJ_Standalone
-        PRIVATE
-                ""
-                $<$<STREQUAL:"${CMAKE_Fortran_COMPILER_ID}","GNU">:-ffree-line-length-none>
-)
```

Comment:
I don't understand why, but with these cmake ended up with a bad makefile can couldn't continue. Just linking the standalone to `CloudJ_Core` worked

Reply:
I do not understand why this would cause problems either. The update seems to be fine since I get zero diff using before and after on my system. Bonus points for fewer lines of code.